docs/data/external-oasis-repos.md

```diff
@@ -1,60 +1,79 @@
+!!! warning
+    RHEL7 only supports Overlay-FS if the underlying filesystem is `ext3` or `ext4`; make sure `/var/spool/cvmfs`
+    is one of these filesystem types.
 
-Requirements: setting up a CVMFS repository server
---------------------------------------------------
+    If this is not possible, add `CVMFS_DONT_CHECK_OVERLAYFS_VERSION=yes` to your CVMFS configuration.  Using
+    `xfs` will work if it was created with `ftype=1`
 
-This document doesn't cover requirements for the GOC computers since those are already set up. The requirements for a CVMFS repository server are that it have installed cvmfs-server and cvmfs client, both version 2.2.2 or greater; version 2.3.3 or greater is recommended on EL6 and required on EL7. This requires using aufs or OverlayFS: aufs requires a modified kernel from CERN for Redhat EL6-based systems, and OverlayFS requires at least EL7.3; the latter uses a standard kernel so it is highly recommended. Also an apache httpd server needs to be enabled. NOTE: multiple cvmfs repositories can be hosted on the same machine.
+Installation
+------------
 
-`/srv/cvmfs` will hold all the published data, so make sure it is large enough to hold all the repositories hosted on the machine. `/var/spool/cvmfs` will hold files during publish, so it should be large enough to hold the most amount of data that might be attempted to be published at once. In addition, on EL7 and cvmfs-server-2.4.1, `/var/spool/cvmfs` needs to be of filesystem type 'ext3' or 'ext4', or the `cvmfs_server` command will not allow it. There is a variable to override the filesystem type check, but other filesystem types are not guaranteed to work (xfs created with `ftype=1` should work with `CVMFS_DONT_CHECK_OVERLAYFS_VERSION=yes`).
+### RHEL7-based Systems
 
-As with all OSG software installations, there are some one-time (per host) steps to prepare in advance:
+For a RHEL7-based system, installation is a straightforward install via `yum`:
 
-- Ensure the host has [a supported operating system](../release/supported_platforms)
-- Obtain root access to the host
-- Prepare the [required Yum repositories](../common/yum)
+``` console
+root@host # yum install cvmfs-server osg-oasis 
+```
+
+### RHEL6-based Systems
 
-This is the procedure for installing on a Redhat EL6-based system:
+A RHEL6 host needs additional steps in order to add the AUFS2 kernel module.
 
 ``` console
-root@host # rpm -i https://cvmrepo.web.cern.ch/cvmrepo/yum/cvmfs-release-latest.noarch.rpm 
-root@host # yum install --enablerepo=cernvm-kernel --disablerepo=cernvm kernel aufs2-util cvmfs-server.x86_64 cvmfs.x86_64 cvmfs-config-osg 
-root@host # echo "cvmfs_server mount -a" >>/etc/rc.local 
+root@host # rpm -i https://cvmrepo.web.cern.ch/cvmrepo/yum/cvmfs-release-latest.noarch.rpm
+root@host # yum install --enablerepo=cernvm-kernel --disablerepo=cernvm kernel aufs2-util cvmfs-server.x86_64 cvmfs.x86_64 cvmfs-config-osg
```

Comment:
It occurs to me that "cvmfs.x86_64 cvmfs-config-osg" should change to "osg-oasis".